javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -364,6 +364,7 @@ export async function generateNetworkSpec(
           parachain.register_para === undefined
             ? true
             : parachain.register_para, // register by default
+        onboardAsParachain: parachain.onboard_as_parachain || false,
```

Comment:
@antonva this shouldn't be `true` by default to keep the actual behavior (register as parachain)?

Reply:
Oh yeah, I missed the change there. Thanks!